### ISSUE TITLE
Fetch from Lodestone failing

### ISSUE THREAD

**Issue by zbee** [Koenari/HimbeertoniRaidTool#151]
I have tried searching for all the characters in my static, but each time it fails.

Here's the logs for the attempt on me, with a couple extra lines of logging added in the middle:

```
13:40:16.775 | INF | [HimbeertoniRaidTool] Started gear update for Radaryn Saast
13:40:16.777 | INF | [HimbeertoniRaidTool] Using name and home world to search...
13:40:16.777 | INF | [HimbeertoniRaidTool] (Name: "Radaryn Saast", World: "Goblin")
13:40:16.785 | INF | [HimbeertoniRaidTool] (netStoneResponse is: null)
13:40:16.786 | WRN | [HimbeertoniRaidTool] Tried parsing LodestoneID but failed.
13:40:16.786 | WRN | [HimbeertoniRaidTool] Character not found on Lodestone.
13:40:17.277 | ERR | [HimbeertoniRaidTool] Task "Update Radaryn Saast from Lodestone" finished with an error: Character not found on Lodestone.
```

<details>
<summary>And this is my character in the DB</summary>

```json
{
    "Classes": [
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 138,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 139,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 20,
        "Level": 90
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 167,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 166,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": false,
        "Job": 32,
        "Level": 100
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 171,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 168,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": false,
        "Job": 33,
        "Level": 100
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 172,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 169,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 24,
        "Level": 100
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 173,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 170,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 30,
        "Level": 90
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 177,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 176,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 22,
        "Level": 90
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 179,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 178,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": false,
        "Job": 21,
        "Level": 100
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 186,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 182,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 40,
        "Level": 73
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 187,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 183,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 38,
        "Level": 60
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 188,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 184,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 26,
        "Level": 1
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 189,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 185,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 36,
        "Level": 80
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 191,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 190,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": false,
        "Job": 35,
        "Level": 100
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 193,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 192,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 25,
        "Level": 90
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 195,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 194,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 19,
        "Level": 90
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 198,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 196,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 34,
        "Level": 90
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 199,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 197,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 39,
        "Level": 90
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 201,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 200,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 31,
        "Level": 90
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 203,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 202,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": false,
        "Job": 23,
        "Level": 90
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 205,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 204,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 41,
        "Level": 90
      },
      {
        "BisSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 207,
            "Signature": ""
          }
        ],
        "GearSets": [
          {
            "Revision": 1,
            "Authority": 351723681,
            "Type": 3,
            "Sequence": 206,
            "Signature": ""
          }
        ],
        "ActiveBiSIdx": 0,
        "ActiveGearIndex": 0,
        "Hide": true,
        "Job": 37,
        "Level": 90
      }
    ],
    "MainInventory": {},
    "RemoteIDs": [],
    "Wallet": {
      "Gil": 8545884,
      "TomestoneOfComedy": 1406,
      "TomestoneOfCausality": 1440
    },
    "MainJob": 32,
    "HrtCharID": 9053766040135378976,
    "Gender": 0,
    "WorldID": 81,
    "LodestoneID": 0,
    "Name": "Radaryn Saast",
    "Tribe": 16,
    "LocalID": {
      "Revision": 1,
      "Authority": 351723681,
      "Type": 2,
      "Sequence": 15,
      "Signature": ""
    }
  }
```

</details>

---

I assume this is an error in [xivapi/NetStone](https://github.com/xivapi/NetStone), but I wasn't able to verify that, and it results in a usability issue here anyway.

**Comment by zbee** [comment]
If I manually add the Lodestone ID in the JSON, I get these logs:
```
13:51:08.140 | INF | [HimbeertoniRaidTool] Started gear update for Radaryn Saast
13:51:08.140 | INF | [HimbeertoniRaidTool] Using ID to search...
13:51:08.140 | INF | [HimbeertoniRaidTool] (ID: "25653876")
13:51:08.141 | WRN | [HimbeertoniRaidTool] Character not found on Lodestone.
13:51:08.544 | ERR | [HimbeertoniRaidTool] Task "Update Radaryn Saast from Lodestone" finished with an error: Character not found on Lodestone.
```

**Comment by Koenari** [comment]
Fixed in 1.6.2.7
